lib/readthis/cache.rb

```diff
@@ -35,6 +35,9 @@ def self.notifications
     #
     def initialize(options = {})
       @options = options
+      if @options[:expires_in] and @options[:expires_in].respond_to?(:to_i)
+        @options[:expires_in] = @options[:expires_in].to_i
```

Comment:
This won't be applied to option overrides, as you fixed in the other PR recently. That inconsistency would certainly be confusing.